s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3WireMockBase.scala

```diff
@@ -955,8 +955,8 @@ object S3WireMockBase {
     |  path-style-access = false
     |  endpoint-url = "http://localhost:$proxyPort"
     |  retry-settings {
-    |    min-backoff = 0s
-    |    max-backoff = 0s
+    |    min-backoff = 100ms
+    |    max-backoff = 300ms
```

Comment:
Was this intentionally part of this PR?

Reply:
Oops, my bad... forgot to rebase after the last PR was merged.  Fixed.